optaplanner-examples/src/main/java/org/optaplanner/examples/meetingscheduling/persistence/MeetingSchedulingXlsxFileIO.java

```diff
@@ -0,0 +1,722 @@
+        }
+
+        public Workbook write() {
+            workbook = new XSSFWorkbook();
+            creationHelper = workbook.getCreationHelper();
+            createStyles();
+            writeConfiguration();
+            writePersons();
+            writeMeetings();
+            writeDays();
+            writeRooms();
+            writeRoomsView();
+            writePersonsView();
+
+            return workbook;
+        }
+
+        private void writeConfiguration() {
+            nextSheet("Configuration", 1, 3, false);
+            nextRow();
+            nextHeaderCell("Meeting name");
+            nextCell().setCellValue(""); // TODO: add a meetingName field to class MeetingSchedule?
+            nextRow();
+            nextRow();
+            nextHeaderCell("Constraint");
+            nextHeaderCell("Weight");
+            nextHeaderCell("Description");
+
+            writeConstraintLine(DO_ALL_MEETINGS_AS_SOON_AS_POSSIBLE, null, "");
+            nextRow();
+            writeConstraintLine(REQUIRED_AND_PREFERRED_ATTENDANCE_CONFLICT, null, "");
+            writeConstraintLine(PREFERRED_ATTENDANCE_CONFLICT, null, "");
+            nextRow();
+            writeConstraintLine(ROOM_CONFLICT, null, "");
+            writeConstraintLine(DONT_GO_IN_OVERTIME, null, "");
+            writeConstraintLine(REQUIRED_ATTENDANCE_CONFLICT, null, "");
+            writeConstraintLine(REQUIRED_ROOM_CAPACITY, null, "");
+            writeConstraintLine(START_AND_END_ON_SAME_DAY, null, "");
+            autoSizeColumnsWithHeader();
+        }
+
+        private void writeConstraintLine(String name, Supplier<Integer> supplier, String constraintDescription) {
+            nextRow();
+            nextHeaderCell(name);
+            nextCell().setCellValue("n/a");
+            nextHeaderCell(constraintDescription);
+        }
+
+        private void writePersons() {
+            nextSheet("Persons", 1, 0, false);
+            nextRow();
+            nextHeaderCell("Full name");
+            for (Person person : solution.getPersonList()) {
+                nextRow();
+                nextCell().setCellValue(person.getFullName());
+            }
+            autoSizeColumnsWithHeader();
+        }
+
+        private void writeMeetings() {
+            nextSheet("Meetings", 2, 1, false);
+            nextRow();
+            nextHeaderCell("Topic");
+            nextHeaderCell("Duration");
+            nextHeaderCell("Speakers");
+            nextHeaderCell("Content");
+            nextHeaderCell("Required attendance list");
+            nextHeaderCell("Preferred attendance list");
+            for (Meeting meeting : solution.getMeetingList()) {
+                nextRow();
+                nextCell().setCellValue(meeting.getTopic());
+                nextCell().setCellValue(meeting.getDurationInGrains() * TimeGrain.GRAIN_LENGTH_IN_MINUTES);
+                nextCell().setCellValue(meeting.getSpeakerList() == null ? "" :
```

Comment:
Would be nice if `getSpeakerList()` never returned null so we don't need to null-check.

Reply:
@ge0ffrey We're checking for null values very often, this requires modifying all the getters in the domain. While I agree with @yurloc I think it's better to do it in a separate PR and do it for all the examples to keep be consistent.

Reply:
+1 for doing this separately (and optionally).